tests/js/server/aql/aql-traversal-graph-object-regression.js

```diff
@@ -0,0 +1,158 @@
+/*jshint globalstrict:false, strict:false, maxlen: 500 */
+/*global assertEqual, AQL_EXECUTE, assertTrue, fail */
+
+////////////////////////////////////////////////////////////////////////////////
+/// @brief tests for regression returning blocks to the manager
+///
+/// @file
+///
+/// DISCLAIMER
+///
+/// Copyright 2010-2014 triagens GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is triAGENS GmbH, Cologne, Germany
+///
+/// @author Markus Pfeiffer
+/// @author Copyright 2020, triAGENS GmbH, Cologne, Germany
+////////////////////////////////////////////////////////////////////////////////
+
+var jsunity = require("jsunity");
+var internal = require("internal");
+var errors = internal.errors;
+var db = require("@arangodb").db, indexId;
+
+const vertexCollectionName = "person";
+const edgeCollectionName = "link";
+
+var cleanup = function() {
+  db._drop(vertexCollectionName);
+  db._drop(edgeCollectionName);
+};
+
+var createBaseGraph = function () {
+  const vc = db._create(vertexCollectionName, { numberOfShards: 4 });
+  const ec = db._createEdgeCollection(edgeCollectionName, { numberOfShards: 4 });
```

Comment:
If that is cluster-only test file should have the following name: `aql-traversal-graph-object-regression-cluster.js`. Otherwise `numberOfShards` should be omitted.

Reply:
`numberOfShards` can also be specified on single server, but there it will be ignored.
It helps specifying it in general tests, because then the test file can be used in both single server and cluster cases, but in the cluster we make sure we have more than just a single shard (which often hid problems due to different shards being involved).